docs/index.rst

```diff
@@ -7,6 +7,7 @@ Welcome to Ape's documentation!
    :maxdepth: 2
 
    quickstart
+   plugins
```

Comment:
Looks like your docs changes are in this branch!

You should make this branch only contain the `plugins list` related work.

Or create a new branch from master and just copy in the changes, whatever is easier.

Reply:
i am chatting with him about this right now, do you know the best way to move his new changes to the new branch?

Reply:
i was thinking he could `git reset HEAD~numberofcommits` and then stash those changes? then pop the stash in the new branch?

Reply:
I didn't know you could pop stash changes in different branches then when you stashed/

I would just create clean branch off of `main` and copy in the `_cli.py` changes... You are only changing one file so that should be simple enough, and there is less chance of something going wrong.

Reply:
I see what you are saying now. but if you reset you may lose your current work here? IDK. Undoing everything and pasting it back in from the beginning is still the least anxious way lol